src/Runtime.Base/src/System/Runtime/InternalCalls.cs

```diff
@@ -190,6 +191,17 @@ internal static int RhEndNoGCRegion()
         [ManuallyManaged(GcPollPolicy.Never)]
         internal extern static unsafe void RhpAssignRef(ref object address, object obj);
 
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        [RuntimeImport(Redhawk.BaseName, "RhpInitMultibyte")]
+        internal static extern unsafe void RhpInitMultibyte(ref byte dmem, int c, nuint size);
+
+        [DllImport(Redhawk.BaseName)]
```

Comment:
Would it be okay to make this RuntimeImport?

Reply:
I think so. 

RuntimeImport is a bit problematic for GC suspension, but we have same shortcut in many other places. The right solution is to make RuntimeImport GC suspension friendly by teaching JIT how to probe for GC around it efficiently.

Reply:
This is clone of the memmove from the CoreLib. The one in CoreLib needs to stay as `DllImport`. Skipping GC probing around that one would be too impactful.